infer.py

```diff
@@ -1,58 +1,78 @@
+import os
+import tqdm
+import json
 import argparse
 import torch
+import functools
 import models
 import shapes
 import audio
+import vad
 import transcripts
+import multiprocessing as mp
+
+
+def get_transcript(model, audio_path: str, sample_rate: int):
+	signal: shapes.BT; sample_rate: int
+	signal, sample_rate = audio.read_audio(audio_path, sample_rate=sample_rate, dtype=model.input_dtype, mono=True, __array_wrap__=torch.as_tensor)
+
+	speaker_mask = model.get_speaker_mask(signal, sample_rate)
+
+	return dict(
+		audio_path = audio_path,
+		audio_name = os.path.basename(audio_path),
+		intervals = transcripts.mask_to_intervals(speaker_mask, sample_rate),
```

Comment:
i think we decided to convert this to the format used in convast, i.e. falt list of dicts

Reply:
rename `transcripts.mask_to_intervals` -> `transcripts.mask_to_transcript`  transcript is flat list of dicts